src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -2498,6 +2499,7 @@ public async Task TestCreateWithBufferNotInWorkspace()
                 workspace.GetService<ClassificationTypeMap>(),
                 globalOptions,
                 visibilityTracker: null,
+                workspace.GetService<TaggerThreadCoordinator>(),
```

Comment:
a ton of changes are just mef importing hte helper type and passing it down through to the tagger itself.